tests/jeos/firstrun.pm

```diff
@@ -70,10 +70,22 @@ sub run {
 
     # JeOS on generalhw
     mouse_hide;
+
+    # attach serial console to active VNC on z/kvm host
+    # in order to interact with the firstboot wizard
+    my $con;
+    my $welcome_to = 300;
+    if (is_s390x && is_svirt) {
+        $con = select_console('svirt', await_console => 0);
+        my $name = $con->name;
+        enter_cmd("virsh console --devname console0 --force $name");
+        $welcome_to = 420;
```

Comment:
I wonder why it takes > 5 min here

Reply:
There is a long timeout for the `ignition|combustion` services.

```
Performing 'systemctl kexec' (just-in-case) Running in chroot, ignoring command 'kexec'
Performing 'kexec -e -x'[    2.838212] dracut-pre-udev[434]: sh: /sbin/sysctl: No such file or directory
[    2.838880] dracut-pre-udev[433]: modprobe: ERROR: Error running install command '/sbin/modprobe --ignore-install sunrpc  && /sbin/sysctl -q --pattern sunrpc --system' for module sunrpc: retcode 127
[    2.838928] dracut-pre-udev[433]: modprobe: ERROR: could not insert 'sunrpc': Invalid argument
[ TIME ] Timed out waiting for device /dev/disk/by-label/ignition.
[ TIME ] Timed out waiting for device /dev/combustion/config.
[  312.514266] dracut-pre-mount[631]: Created symlink /etc/systemd/system/systemd-fsck-root.service → /dev/null.
[  313.891490] dracut-pre-mount[904]: Removed /etc/systemd/system/systemd-fsck-root.service.
```

Reply:
Looks like that's a side effect of ignition's custom qemu_blockdev module: https://bugzilla.suse.com/show_bug.cgi?id=1207679

It adds a 5min timeout. Can you file a bug report?

Reply:
Yes, it was mentioned by Ignaz in https://bugzilla.suse.com/show_bug.cgi?id=1207679#c10 as a better solution should come up in the future (hopefully).
Do you think it still deserves a specific bug report only for the timeout?

Reply:
IMO yes. 5 min of delay for a feature we don't need is a bug.

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=1210429